core/server/helpers/tpl/subscribe_form.hbs

```diff
@@ -1,11 +1,11 @@
-<form method="post" action="{{action}}" class="{{form_class}}">
+<form method="post" action="{{action}}" id={{form_id}} class="{{form_class}}">
```

Comment:
PR looks good to me :) Works great 👍 

I just wondered

> id={{form_id}} class="{{form_class}}"

We use double slashes and no double slashes. Looks like it doesn't matter? But maybe we should use double slashes for consistency reasons?